packages/i18n/README.md

```diff
@@ -0,0 +1,198 @@
+# I18n Package
+
+This package provides a set of tools to help you internationalize your Chrome Extension.
+
+https://developer.chrome.com/docs/extensions/reference/api/i18n
+
+## Installation
+
+If you want to use the i18n translation function in each pages, you need to add the following to the package.json file.
+
+```json
+{
+  "dependencies": {
+    "@extension/i18n": "workspace:*"
+  }
+}
+```
+
+Then run the following command to install the package.
+
+```bash
+pnpm install
+```
+
+## Manage translations
+
+You can manage translations in the `locales` directory.
+
+`locales/en/messages.json`
+
+```json
+{
+  "helloWorld": {
+    "message": "Hello, World!"
+  }
+}
+```
+
+`locales/ko/messages.json`
+
+```json
+{
+  "helloWorld": {
+    "message": "안녕하세요, 여러분!"
+  }
+}
+```
+
+If you want to use placeholders, you can use the following format.
+
+https://developer.chrome.com/docs/extensions/how-to/ui/localization-message-formats#placeholders
+
+> [!Caution]
+> You should use placeholders in this format: `$1`, `$2`, `$3`, ... $/d+
+> Still, We don't support the placeholders field in the `messages.json` file.
+
+`locales/en/messages.json`
+
+```json
+{
+  "helloAnd": {
+    "message": "Hello, $1 $2"
```

Comment:
I don't understand how to use this placeholders in json i have this $1, as you have described here, but nothing was showed.

OOO now i got it.
```ts
console.log(t('hello', ['hello', 'xd']));
```

I should define array after.
I think good idea could be, to add example like this in this README, because of this developer.chrome example isn't good enough in this case.